company.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <string.h>
+
+typedef struct employee {
+	char name[100];
+	char pos[100];
+	char pay[100];
```

Comment:
Nicely done! the only thing is it would have been nice to see you used mixed data types (so a float for the pay instead). Also, again, more comments!